config/rbac/csi_rbd_nodeplugin_role.yaml

```diff
@@ -1,7 +1,7 @@
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: csi-rbd-nodeplugin-r
+  name: csi-rbd-nodeplugin-role
```

Comment:
The `-r` suffix was chosen on purpose. Why change it?

Reply:
done.